Source/common/SNTConfigurator.m

```diff
@@ -259,12 +260,22 @@ - (NSString *)syncServerAuthRootsFile {
   return self.configData[kServerAuthRootsFileKey];
 }
 
-- (NSDate *)syncLastSuccess {
-  return self.configData[kSyncLastSuccess];
+- (NSDate *)fullSyncLastSuccess {
+  return self.configData[kFullSyncLastSuccess];
 }
 
-- (void)setSyncLastSuccess:(NSDate *)syncLastSuccess {
-  self.configData[kSyncLastSuccess] = syncLastSuccess;
+- (NSDate *)ruleSyncLastSuccess {
```

Comment:
Swap the position of this with setFullSyncLastSuccess so the related getters/setters are kept together